meshtastic/config.proto

```diff
@@ -1006,8 +1009,57 @@ message Config {
 
     /*
      * Enables device (serial style logs) over Bluetooth
+     * Moved to SecurityConfig
      */
-    bool device_logging_enabled = 4;
+    bool device_logging_enabled = 4[deprecated = true];
+  }
+
+  message SecurityConfig {
+
+  /*
+   * The public key of the user's device.
+   * This is sent out to other nodes on the mesh to allow them to compute a shared secret key.
+   */
+    bytes public_key = 1;
+
+  /*
+   * The private key of the device.
+   * This is used to create a shared key with a remote device.
+   */
+    bytes private_key = 2;
+
+    /*
+     * This is the public key authorized to send admin messages to this node
+     */
+    bytes admin_key = 3;
+
+    /*
+     * If true, device is considered to be "managed" by a mesh administrator
+     * Clients should then limit available configuration and administrative options inside the user interface
+     */
+    bool is_managed = 4;
+
+    /*
+     * Disabling this will disable the SerialConsole by not initilizing the StreamAPI
+     */
+    bool serial_enabled = 5;
+
+    /*
+     * By default we turn off logging as soon as an API client connects (to keep shared serial link quiet).
+     * Set this to true to leave the debug log outputting even when API is active.
+     */
+    bool debug_log_enabled = 6;
+
+    /*
+     * Enables device (serial style logs) over Bluetooth
+     * Moved to SecurityConfig
+     */
+    bool bluetooth_logging_enabled = 7;
```

Comment:
@jp-bennett @garthvh I'm waffling on this one again.... I think debug_log_enabled makes sense because serial is not protected, but bluetooth is pin protected. If someone gains access to bluetooth, they have everything anyway.